sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ListModelsAsync.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Based on our discussion for samples in the morning. Recommend only keeping the sync sample.

Reply:
I feel list models() need to have async samples. It is like listConfigurationSetting(). Otherwise, user won't know how to use aysnc listModels()

Reply:
We can have this additional sample too. It should help since it will show async way of iteration through the paged response.